/**:
  ros__parameters:
    distance_threshold: 0.5
    use_dynamic_map_loading: true
    downsize_ratio_z_axis: 0.5
    timer_interval_ms: 100
    map_update_distance_threshold: 10.0
    map_loader_radius: 150.0
    publish_debug_pcd: False
